packages/base/image.gts

```diff
@@ -0,0 +1,46 @@
+import { CardDef, Component, contains, field } from './card-api';
+import UrlField from './url';
+
+class ImageCardView extends Component<typeof ImageCard> {
+  <template>
+    {{#if @model.url}}
+      <img
+        class='image-card__media'
+        src={{@model.url}}
+        alt=''
+        loading='lazy'
+      />
+    {{else}}
+      <div class='image-card__placeholder'>No image URL provided</div>
+    {{/if}}
+    <style scoped>
+      .image-card__media {
```

Comment:
With scoping, is there any need for this BEM-esque nomenclature?

Reply:
Nope